src/backend/joanie/tests/core/test_api_course.py

```diff
@@ -314,7 +314,7 @@ def test_api_course_read_detail_authenticated(self):
 
         with self.assertNumQueries(36):
             response = self.client.get(
-                f"/api/courses/{course.code}/",
+                reverse("courses-detail", kwargs={"code": course.code}),
```

Comment:
I wouldn't use reverse in tests for 2 reasons:
- we want to test the url we think it should be and be informed if it changes inadvertently! 
- it is harder to understand which url you're calling for the developper reading the test